src/node_osrm_support.hpp

```diff
@@ -733,6 +733,74 @@ argumentsToTripParameter(const Nan::FunctionCallbackInfo<v8::Value> &args,
         return trip_parameters_ptr();
     }
 
+    if (obj->Has(Nan::New("roundtrip").ToLocalChecked()))
+    {
+        auto roundtrip = obj->Get(Nan::New("roundtrip").ToLocalChecked());
```

Comment:
Here and below: what you can do instead of `Has()`+`Get()` is the following:

```c++
auto maybeRoundtrip = Nan::Get(..);
if (maybeRoundtrip.empty() || !maybeRoundtrip.ToLocalChecked()->IsBoolean) error();
```

`Nan::Get` returns you a Maybe type similar to `boost::optional<T>`.